pages/learn/develop/integrations/cloud-iot-provisioning.md

```diff
@@ -0,0 +1,45 @@
+---
+dynamic:
+  variables: [ $cloud ]
+  ref: $original_ref/$cloud
+  $switch_text: Cloud IoT Provisioning for $cloud
```

Comment:
```suggestion
  $switch_text: I want to provision Cloud IoT device for $cloud
```

Reply:
Good idea. I see how your wording is parallel to that in the Get Started section for a device.

Reply:
Fixed in 771cf24.